pkg/reconciler/app/controller.go

```diff
@@ -40,6 +47,25 @@ func NewController(ctx context.Context, cmw configmap.Watcher) *controller.Impl
 	sourceInformer := sourceinformer.Get(ctx)
 	appInformer := appinformer.Get(ctx)
 	spaceInformer := spaceinformer.Get(ctx)
+	serviceBindingInformer := servicebindinginformer.Get(ctx)
+
+	// TODO: replace all of this code which eventually gets the systemEnvInjector
```

Comment:
Do we have an issue capturing this?

Reply:
Updated to reference https://github.com/google/kf/issues/397